.rb

```diff
@@ -0,0 +1,47 @@
+puts "Welcome to the ADA Calculator"
```

Comment:
Let me just say you should name this file something **other** than `.rb`.  `calculator.rb` would be a better filename.